contracts/governance/liquidation/LiquidationStrategy.sol

```diff
@@ -0,0 +1,170 @@
+// SPDX-License-Identifier: agpl-3.0
+pragma solidity 0.7.6;
+
+import {PermissionAdmin} from '@kyber.network/utils-sc/contracts/PermissionAdmin.sol';
+import {Utils} from '@kyber.network/utils-sc/contracts/Utils.sol';
+import {IERC20Ext} from '@kyber.network/utils-sc/contracts/IERC20Ext.sol';
+import {ReentrancyGuard} from '@openzeppelin/contracts/utils/ReentrancyGuard.sol';
+import {SafeMath} from '@openzeppelin/contracts/math/SafeMath.sol';
+import {SafeERC20} from '@openzeppelin/contracts/token/ERC20/SafeERC20.sol';
+import {ILiquidationStrategy} from '../../interfaces/ILiquidationStrategy.sol';
+import {ILiquidationCallback} from '../../interfaces/ILiquidationCallback.sol';
+import {ITreasuryPool} from '../../interfaces/ITreasuryPool.sol';
+
+contract LiquidationStrategy is ILiquidationStrategy, PermissionAdmin, Utils, ReentrancyGuard {
+
+  using SafeERC20 for IERC20Ext;
+  using SafeMath for uint256;
+
+  // after repeatedPeriod since startTime, there will be duration (in seconds)
+  // for liquidators to liquidate tokens in the fee pool
+  // for example: from deployed time, every 2 weeks liquidation is enabled for 4 days
+  struct LiquidationSchedule {
+    uint128 startTime;
+    uint64 repeatedPeriod;
+    uint64 duration;
+  }
+
+  LiquidationSchedule public liquidationSchedule;
+  ITreasuryPool public feePool;
```

Comment:
This is confused...
I mean we can have `ITreasuryPool` and `IFeePool`

Reply:
change to IPool and Pool contract, then Fee and Treasury inherit that contract